src/views/templates/details/tabs/scheduling/components/Descheduler.tsx

```diff
@@ -58,22 +63,33 @@ const Descheduler: React.FC<TemplateSchedulingGridProps> = ({ template }) => {
       </DescriptionListTermHelpText>
 
       <DescriptionListDescription>
-        {useDeschedulerInstalled && (
-          <Button
-            isInline
-            isDisabled={!isDeschedulerInstalled}
-            onClick={() =>
-              createModal(({ isOpen, onClose }) => (
-                <DeschedulerModal template={template} isOpen={isOpen} onClose={onClose} />
-              ))
-            }
-            variant="link"
-            iconPosition={'right'}
-          >
-            {isDeschedulerOn ? t('ON') : t('OFF')}
-            <PencilAltIcon className="co-icon-space-l pf-c-button-icon--plain" />
-          </Button>
-        )}
+        <Tooltip
+          content={
+            !isDeschedulerInstalled &&
```

Comment:
you should not have conditional help messages

Reply:
So what is the correct way to display different messages for different scenarios?